tests/models/pixtral/test_modeling_pixtral.py

```diff
@@ -258,35 +253,3 @@ def test_disk_offload_safetensors(self):
     @unittest.skip(reason="Not supported yet")
     def test_determinism(self):
         pass
-
-
-@require_torch
-class PixtralVisionModelIntegrationTest(unittest.TestCase):
```

Comment:
why did this one have to go away ? 😓

Reply:
Yep it needs to be in the `Llava` tests! Can you move it around 👀

Reply:
sure!

Reply:
https://github.com/huggingface/transformers/blob/913330ca9f80b0a308d7490a02274b01b51e6051/tests/models/llava/test_modeling_llava.py#L620-L663 

looks like pixtral is already tested under llava tests properly - since it's a llava model it's enough to test it there, no?

Reply:
ah then good job and sorry!

Reply:
no worries- it's hard to see when tests are measuring a model through another one, but it's convenient on our side. I can move all pixtral related tests to pixtral testing so we don't forget it, wdyt? It's a bit more loc but it's easier to track down